WordPress/Tests/WP/UseCapabilitiesNotRolesUnitTest.inc

```diff
@@ -0,0 +1,41 @@
+<?php
+
```

Comment:
Why the extra lines here?

Reply:
They were there from the original commit and I was too lazy to remove them and update the warning and error list. 🙃

Reply:
You may as well remove them as this is a new sniff.